presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenEvent.kt

```diff
@@ -0,0 +1,7 @@
+package me.androidbox.presentation.event.screen
+
+import android.net.Uri
+
+sealed interface EventScreenEvent {
```

Comment:
Not fully sure whats the use of this

Reply:
Seems like a normal (but incomplete) screen event class to me, or what are you referring to @MKiperszmid?